cfgov/search/backends.py

```diff
@@ -1,10 +1,16 @@
 # Based on https://wellfire.co/learn/custom-haystack-elasticsearch-backend/
 
+import haystack.backends.elasticsearch2_backend
 from django.conf import settings
 from haystack.backends.elasticsearch2_backend import (
     Elasticsearch2SearchBackend, Elasticsearch2SearchEngine
 )
 
+import elasticsearch
+
+
+haystack.backends.elasticsearch2_backend.elasticsearch = elasticsearch
```

Comment:
What is this attempting to do?

Reply:
Not sure what was done here and why so I removed them.

Reply:
I don't know why this was done so I removed the duplicate import of elasticsearch.